### ISSUE TITLE
Rendering issue crash drum notation

### ISSUE THREAD

**Issue by jeroenlammerts** [0xfe/vexflow#752]
```
        let notes = [
          new VF.StaveNote({ keys: ['f/4', 'g/5/x3'], duration: '8' }),
          new VF.StaveNote({ keys: ['f/5/x2'], duration: '8' }),
          new VF.StaveNote({ keys: ['c/5', 'f/5/x2'], duration: '8' }),
          new VF.StaveNote({ keys: ['f/5/x2'], duration: '16' }),
          new VF.StaveNote({ keys: ['f/4'], duration: '16' }),
          new VF.StaveNote({ keys: ['f/4', 'f/5/x2'], duration: '8' }),
          new VF.StaveNote({ keys: ['f/4', 'f/5/x2'], duration: '8' }),
          new VF.StaveNote({ keys: ['c/5', 'f/5/x2'], duration: '8' }),
          new VF.StaveNote({ keys: ['f/5/x2'], duration: '8' }),
        ];

          let voice1 = new VF.Voice({})
            .addTickables(notes)
            .setStave(measure);

          let beams = VF.Beam.generateBeams(voice1.getTickables(), {
            maintain_stem_directions: true,
            flat_beams: true,
          });

          new VF.Formatter()
            .joinVoices([voice1])
            .formatToStave([voice1], measure);

          voice1.setContext(this.ctx).draw();

          beams.forEach(beam => {
            beam.setContext(this.ctx).draw();
          });
```

![Screenshot from 2020-03-28 17-34-42](https://user-images.githubusercontent.com/4282711/77828331-1050ae00-711b-11ea-8022-f05093cfb9bb.png)

The crash should be more to the left.

**Comment by 0xfe** [comment]
Looking into this. More weird cases:

<img width="350" alt="Screen Shot 2020-04-01 at 2 54 28 PM" src="https://user-images.githubusercontent.com/241299/78175377-bf95c980-7428-11ea-8a4a-ce20ee65fb82.png">

**Comment by 0xfe** [comment]
This was a super thorny issue -- took three hours to figure out and fix. Running regression tests now and hope I haven't broken anything in the process.

**Comment by 0xfe** [comment]
Okay, all fixed, however there are still some minor adjustments required on a few glyphs. Whew.

<img width="471" alt="Screen Shot 2020-04-01 at 5 40 18 PM" src="https://user-images.githubusercontent.com/241299/78189392-ef9c9700-743f-11ea-9667-fc2fc80b5713.png">

**Comment by jeroenlammerts** [comment]
Thanks @0xfe for the effort looking into this. Actually the latest release (1.2.91) gives me some error. I've tested it with 1.2.90 which doesn't show the error.

![Screenshot from 2020-04-02 19-48-44](https://user-images.githubusercontent.com/4282711/78281575-2abfc880-751b-11ea-998c-dbf2180e3048.png)
![Screenshot from 2020-04-02 19-45-33](https://user-images.githubusercontent.com/4282711/78281578-2b585f00-751b-11ea-9c63-b12cfd400fb2.png)
![Screenshot from 2020-04-02 19-45-16](https://user-images.githubusercontent.com/4282711/78281579-2b585f00-751b-11ea-87e8-9881a93f1af7.png)

**Comment by 0xfe** [comment]
How are you importing the library? Can you send me reproducible sample please?

**Comment by jeroenlammerts** [comment]
`import * as Vex from 'vexflow';`

but now only works with

`import Vex from 'vexflow';`

**Comment by sschmidTU** [comment]
I can confirm, the OSMD webpack build actually still compiled, but starting with Vexflow 1.2.91, when running the build/webpack server, Vex.Flow was undefined at runtime.
We had to use:
`import Vex from "vexflow";`
instead of:
`import Vex = require("vexflow");`

Also, in Typescript, I had to add this to tsconfig.json (to existing compilerOptions):
```json
  "compilerOptions": {
    "esModuleInterop": true,
  }
```

Without that, this new import syntax was giving compile errors.

Now the build is running fine and looking good!

It's unfortunate we weren't discussing this in a dedicated issue.
